test/TensorFlowRuntime/numpy_conversion.swift

```diff
@@ -52,6 +52,12 @@ NumpyConversionTests.test("shaped-array-conversion") {
                 array)
   }
 
+  let reshaped = np.reshape(numpyArrayInt32, [2, 3])
```

Comment:
```suggestion
  let reshaped = np.reshape(numpyArrayInt32, [2, 3] as TensorShape)
```
This is not using `TensorShape`, which kind of misses the point. Could you make the shape be an explicit `TensorShape`?

Reply:
🤦‍♂️ ok i see what you were saying now.  How is this look? @rxwei

Reply:
Looks great.